graphql/server.go

```diff
@@ -0,0 +1,28 @@
+package graphql
+
+import (
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/99designs/gqlgen/graphql/handler"
+	"github.com/99designs/gqlgen/graphql/playground"
+)
+
+const defaultPort = "8080"
+
+// StartServer starts the graphql server
+func StartServer() {
+	port := os.Getenv("PORT")
+	if port == "" {
+		port = defaultPort
+	}
+
+	srv := handler.NewDefaultServer(NewExecutableSchema(New()))
+
+	http.Handle("/", playground.Handler("GraphQL playground", "/query"))
+	http.Handle("/query", srv)
+
+	log.Printf("connect to http://localhost:%s/ for GraphQL playground", port)
+	log.Fatal(http.ListenAndServe(":"+port, nil))
+}
```

Comment:
I don't think it's viable to have this listening on it's own port (there's not a lot of benefit to putting it on its own port, and it complicates the load balancing architecture, and has some impacts on the complexity of doing deploys and managing system shutdowns.) 

I would add to line 344 of service/service.go: 

```
	app.AddRoute("/graphql/playground").Handler(playground.Handler("GraphQLPlayground", "/query"))
	app.AddRoute("/graphql/").Handler(handler.NewDefaultServer(handler.NewExecutableSchema(handler.New())))
```

You'll also need to add which HTTP verbs the clients use (.Get().Post().Patch().Put()) you could also do some combination of .Method(http.MethodGet) but that's more verbose.

You can also add authentication to these entire handlers (perhaps particularly the playground, which we can workout some way of using the existing authentication mechanisms. Authentication below the level of the handler is perhaps beyond the scope of this project.

Reply:
As a second thought, it might be better to do this code somewhere within service/ui.go:250 which would give you easier access to the permissions architecture, and wouldn't change the URL.

Reply:
`app.AddRoute("/graphql/").Handler(handler.NewDefaultServer(handler.NewExecutableSchema(handler.New())))` does not work because `handler.NewDefaultServer` has a return type of `*handler.Server`

I'm sure we can get it working this way but something will need to be changed. any ideas on that?

Reply:
^sam and i paired on this

Reply:
What was the outcome of this? Also 8080 should not be the default port, since that's the one of the ports used for the web server

Reply:
outcome was that we can and will start the gql server using Sam's suggestion of `app.AddRoute`. This gives us benefits like: 
1. not having to worry about load balancing
2. metrics